CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Roslyn
+
+Guidelines for contributing to the Roslyn repo.
+
+## Submitting Pull Requests
+
+- **DO** ensure submissions pass all Jenkins legs and are merge conflict free.
+- **DO** ensure submissions meet functional and performance expectations, including scenarios for which the team doesn’t yet have open source tests. This means you may be asked to fix and resubmit your pull request against a new open test case if it fails one of these tests.
+- **DO** follow the [.editorconfig](http://editorconfig.org/) settings for each directory. 
+- **DO NOT** submit changes beyond the level of a bug fix without discussing with the team first, or they will likely be declined.
+- **DO NOT** submit unsolicited language features.  All language feature discussion should begin on the [C# language](https://github.com/dotnet/csharplang#discussion) / [VB language](https://github.com/dotnet/vblang) repos.  Once a feature is championed and validated by LDM a developer will be assigned to help begin a prototype on this repo inside a feature branch.
```

Comment:
Generally I understand the desire to be more positive here.  However I think we need a very direct statement of what will happen when we get PRs that don't meet our needs.  In particular language features.  Lacking a statement on our repo that we close unsolicited language changes it ends up falling to me to say it to each individual developer.  I think it's better to be up front about this.